### ISSUE TITLE
Android: Sharing links to Joplin doesn't create a note

### ISSUE THREAD

**Issue by ilikeorangutans** [laurent22/joplin#2896]
TL;DR Links or text shared to Joplin on Android just kind of disappears. 

## Environment

Joplin version: 1.0.324
Platform: Android 
OS specifics: Android 10

## Steps to reproduce

1. open a website in a browser *or* select text, hit _share_
2. Select Joplin in the list of applications
3. Joplin opens, select _new note_ or pick an existing note
4. the shared text/link is nowhere to be seen

## Describe what you expected to happen

What I would expect is that the shared text/link can be either stored as a new note or added to an existing note. 

## Logfile

I'm not sure what the best way is to get the log from android. Nothing however stands out in the log, there doesn't seem to be anything related to sharing in there.

**Comment by Rahulm2310** [comment]
@ilikeorangutans It is not currently possible to share images or to share selected text from any app to Joplin. It is not implemented yet. It is a project idea for GSOC-2020. [Check this out](https://joplinapp.org/gsoc2020/ideas.html#4-sharing-on-mobile)

**Comment by pcause** [comment]
@Rahulm2310  That is noit the problem he describes.  This functionality used to work.  In a browser if you shared to joplin it would create a new note with the USR as content and title.  Broken in recent uypdates

**Comment by scheunengeist** [comment]
Can confirm the issue.

## Environment
Joplin version: android-v1.0.325 (and downward to at least 1.0.320)
Platform: Android
OS specifics: LineageOS 16.0

## Steps to reproduce
1. Share text chunk from any other application to Joplin (e. g. from text editor, OpenKeychain, todo list, etc.)
2. Joplin is opened, initializes and presents default view.
3. No new note is being created.

## Describe what you expected to happen
Expectation: Upon sharing text toward Joplin, a new note should be created and pre-filled with the shared text, enabling the user to select the appropriate notebook and to save the note. This was the default behaviour in versions at least up until android-v1.0.316. I'm not sure about 318, and can't check back because if I try to downgrade to earlier versions, the app won't start due to database incompatibility.

Verification: I tried several applications to share text to Joplin, which all failed. I can share text to other applications with success. Force-closing and re-starting Joplin has no effect. Reboot of the phone has no effect.

## Logfile
If needed, I can provide log lines from a time frame in which I tried to share text to Joplin. However, afaict, they show no errors.

If Rahulm2310 is right, this functionality was removed for some reason. I think this is a rather useful feature for a note taking app - any chance of getting it back?
Thanks!

**Comment by WKenya** [comment]
Can confirm I have this issue also. Been running Brave browser and am unable to share any URLs into the application. I also tried Chrome Browser and it also fails. I'm guessing something with the Share Intent is busted.

Environment
Samsung Galaxy S9+
Joplin version: 1.0.324
Platform: Android
OS specifics: Android 10

**Comment by xplosionmind** [comment]
Same here. It doesn't matter where I am: I click on Joplin in the system sharing menu and I gust get redirected to the app. Nothing happens.

Device: OnePlus 6T
OS: Android 10 (Oxygen OS)
Joplin version: 1.0.324

**Comment by roman-r-m** [comment]
Yeah, it does not work at the moment. I'm working on it here: https://github.com/laurent22/joplin/pull/2870

**Comment by xplosionmind** [comment]
> I'm working on it here: #2870

Thanks!

**Comment by pcause** [comment]
still doesn't seem fixed.  when is the pull going in to the released version

**Comment by tchaffee** [comment]
Thanks for the fix! When will this make it to the released version on our phones?
